### ISSUE TITLE
QA_EXPT-11 & EXPT-12_Procedure_content-is-same

### ISSUE THREAD

**Issue by BSravanthi** [Virtual-Labs/analog-cmos-vlsi-circuit-design-iiith#36]
Defect Description :
In the Procedure page of 11th and 12th experiments of this lab, found that the content is same.

Actual Result :
In the Procedure page of 11th and 12th experiments of this lab, found that the content is same.

Environment :
OS: Windows 7, Ubuntu-16.04,Centos-6
Browsers:Firefox-42.0,Chrome-47.0,chromium-45.0
Bandwidth : 100Mbps
Hardware Configuration:8GBRAM ,
Processor:i5

Attachments:
![qa_oe_avlsi_i33](https://cloud.githubusercontent.com/assets/13479177/26666045/44058032-46bb-11e7-9d43-155d41a9e370.png)

**Comment by BSravanthi** [comment]
#34

**Comment by sravanthimodepu** [comment]
Yes.. The 11th & 12th experiments belongs to the same content but the simulations are different. Found the same from the vlab.co.in page.

**Comment by BSravanthi** [comment]
Need to contact lab developer. The issue will be opened till we get the response.
